src/main/scala/uk/gov/hmrc/ct/computations/CP81Input.scala

```diff
@@ -16,9 +16,14 @@
 
 package uk.gov.hmrc.ct.computations
 
-import uk.gov.hmrc.ct.box.{CtBoxIdentifier, CtOptionalInteger, Input}
+import uk.gov.hmrc.ct.box.{ValidatableBox, CtBoxIdentifier, CtOptionalInteger, Input}
+import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
 
-case class CP81Input(value: Option[Int]) extends CtBoxIdentifier(name = "Expenditure qualifying for first year allowance (FYA)") with CtOptionalInteger with Input
+case class CP81Input(value: Option[Int]) extends CtBoxIdentifier(name = "Expenditure qualifying for first year allowance (FYA)") with CtOptionalInteger with Input with ValidatableBox[ComputationsBoxRetriever] {
```

Comment:
Its the least of our worries right now TBH ;-), Ill sort that after a) Ive finished the boxes conversion and b) I've done the changes needed for the machinery and plant.